packages/localize/src/LocalizeMixin.js

```diff
@@ -38,6 +37,12 @@ const LocalizeMixinImplementation = superclass =>
           this.__localizeOnLocaleChanged(event);
         };
 
+      this.__boundLocalizeOnLocaleChanging =
```

Comment:
Is there a difference with `this.__boundLocalizeOnLocaleChanging = this.__localizeOnLocaleChanging.bind(this)`?
(or `this.__localizeOnLocaleChanging = this.__localizeOnLocaleChanging.bind(this)` to reduce complexity (and already high) mental load)?

Reply:
Probably identical, I just copied from the bound locale changed method above 😅 so it's at least legible in the sense that they do the same thing. I suppose I could have changed them both in retrospect 🤷